tools/developer/run.sh

```
@@ -105,14 +105,16 @@
-        update-alternatives --set gcc "/usr/bin/gcc-$1"
+        export CC="/usr/bin/gcc-$1"
+        export CXX="/usr/bin/g++-$1"
     fi
 }
 
 function build_doc {
     pushd build > /dev/null || exit 1
     #rm -rf doc/* ; rm -rf locale/*/*/*.mo
-    rm -rf doc/*
+    # Clean only generated files while preserving custom content
+    find doc -type f -name "*.html" -o -name "*.pdf" -delete
```

Shellcheck (0.10.0)

[warning] This action ignores everything before the -o. Use \( \) to group.

(SC2146)